service/src/main/java/bio/terra/pipelines/app/controller/PipelinesApiController.java

```diff
@@ -119,17 +120,20 @@ public ResponseEntity<ApiCreateJobResponse> createJob(
       @PathVariable("pipelineId") String pipelineId, @RequestBody ApiCreateJobRequestBody body) {
     final SamUser userRequest = getAuthenticatedInfo();
     String userId = userRequest.getSubjectId();
+    String jobIdString = body.getJobControl().getId();
+    UUID jobId = validateJobId(jobIdString);
```

Comment:
what do we think abotu using the openapi yaml to validate this for us.  something like what is found in  https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.1.md#support-for-x-www-form-urlencoded-request-bodies

Reply:
nice, I didn't see the "format: uuid" option. that throws a different error that we have to handle differently in the GlobalExceptionHandler. I've implemented it... see what you think.